tests/schema/test_interface.py

```diff
@@ -161,3 +162,27 @@ def anime(self) -> Anime:
 
     assert not result.errors
     assert result.data == {"anime": {"name": "One Piece"}}
+
+
+def test_input_cannot_inherit_from_interface():
```

Comment:
Can these tests cover the actual messages of the errors as well?

Reply:
Do yo know how to catch the exception message with pytest? Or I need to try, catch, save, raise, assert?

Reply:
You can use 
```python
with pytest.raises(ExceptionType) as exc:
    ...
assert exc.value == "something"
```

Reply:
Also, note that `pytest.raises` takes a `match` argument that may be enough here

Reply:
Let me have a look, thanks